app/models/state_file_az_intake.rb

```diff
@@ -77,6 +77,8 @@ class StateFileAzIntake < StateFileBaseIntake
   enum eligibility_married_filing_separately: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_married_filing_separately
   enum eligibility_529_for_non_qual_expense: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_529_for_non_qual_expense
 
+  devise :timeoutable, :timeout_in => 15.minutes
```

Comment:
Could we set this config on the base intake instead?

Reply:
Good call